data_adapter_oemof/build_datapackage.py

```diff
@@ -271,10 +278,24 @@ def save_datapackage_to_csv(self, destination: str) -> None:
         None if the Datapackage has been saved correctly (no checks implemented)
 
         """
+
+        # check if datapackage already has defined its destination
+        if location_to_save_to:
+            pass
+        elif self.location_to_save_to:
+            location_to_save_to = self.location_to_save_to
+        else:
+            raise ValueError(
+                "Please state location_to_save_to either in datapackage or saving call"
+            )
+
+        if tsam:
+            location_to_save_to = location_to_save_to + "_tsam"
```

Comment:
I would leave this up to the user - might be confusing if you set location path and it is saved to different path internally.